src/utils/get-site.mjs

```diff
@@ -0,0 +1,11 @@
+import { error } from './command-helpers.mjs'
+
+export const getSiteByName = async (api, siteName) => {
+  try {
+    const sites = await api.listSites({ name: siteName, filter: 'all' })
+    const siteFoundByName = sites.find((filteredSite) => filteredSite.name === siteName)
+    return siteFoundByName
+  } catch {
+    error('Site not found. Please rerun "netlify link"')
```

Comment:
If we don't find the site because the name doesn't match, then `siteFoundByName` will be `undefined` and this message won't be shown. Is that intended?

Reply:
Ah good call! I've added a throw Error if siteFoundByName is undefined